src/lib/getters.ts

```diff
@@ -112,7 +112,7 @@ export function getVolumeSliderController() {
 
 export function getProgressBarList() {
   return document.querySelector(
-    `[id="${getCurrentId()}"] > div.short-video-container > #player-container > #progress-bar > ytd-progress-bar-line > #progress-bar-line`,
+    `[id="${getCurrentId()}"] > div.short-video-container #progress-bar-line`,
```

Comment:
```suggestion
    `#${getCurrentId()} > div.short-video-container #progress-bar-line`,
```

Reply:
another question is: in the .short-video-container class selector required? typically I would prefer to avoid using class selectors for finding specific elements. potentially whole thing could simpler and less brittle if we made the selector this:
```suggestion
    `#${getCurrentId()} #progress-bar-line`,
```

Reply:
Turning `[id="${getCurrentId()}"]` into `#${getCurrentId()}`, didn't work. I updated the PR with some changes based on your suggestions.

Reply:
ah the #id thing didnt work because the id in this case is a number and query selector doesnt work with numerical ids. to make it work that way you'd have to do `#${CSS.escape(getCurrentId())}` which is worse than `[id="${getCurrentId()}"]` so the way that you have it now will be great!